### DISCUSSION TITLE
has python3.9 support been added

### DISCUSSION THREAD

**Discussion by lukehodges** [Rapptz/discord.py#6301] [Q&A]
hi sorry to bother any people on this forum but i am unsure if python3.9 has been fully supported as the GitHub looks like the latest version is python3.8

**Comment by Harmon758** [comment]
Yes, [discord.py's dependencies have been updated to support Python 3.9](https://github.com/Rapptz/discord.py/commit/7f3737fcc2fc32078b0904131adcaa75d5eda942) in [v1.6.0](https://discordpy.readthedocs.io/en/latest/whats_new.html#v1-6-0).

**Comment by Rapptz** [comment]
3.9 has always been supported, it just required build tools to get the dependencies. The dependencies have been updated so those in 3.9 will get wheels which do not require build tools.
